app/views/admin/inbound_api_configurations/index.haml

```diff
@@ -0,0 +1,43 @@
+- content_for(:title, t('.title'))
```

Comment:
fyi its showing up as just "Title"

Reply:
Didn't catch that, but @yakloinsteak it should be something like:
```
- title = _('Street 2 Home')
- content_for :title, title
```

Reply:
I copied the mistake from another file, but I have it fixed locally already.